seed/data_importer/meters_parser.py

```diff
@@ -201,44 +244,43 @@ def _parse_gb_meter_details(self):
 
     def _get_property_id(self, source_id, shared_details):
         """
-        Find and cache property_ids to avoid querying for the same property_id
```

Comment:
The biggest change for this method was the shift from finding one (the first) property_id to capturing all property_ids associated to PropertyStates with multiple property_ids. This shift affects downstream methods.